batch/create-batch-tables.sql

```diff
@@ -38,20 +38,21 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   `input_log_uri` VARCHAR(1024),
   `main_log_uri` VARCHAR(1024),
   `output_log_uri` VARCHAR(1024),
-  PRIMARY KEY (`id`),
+  PRIMARY KEY (`batch_id`, `job_id`),
   FOREIGN KEY (`batch_id`) REFERENCES batch(id) ON DELETE CASCADE
 ) ENGINE = InnoDB;
 CREATE INDEX jobs_user ON jobs (user);
 CREATE INDEX jobs_batch ON jobs (batch_id);
```

Comment:
this index is completely superseded, AFIAK, by the primary key. It's possible I don't understand postgres well, but usually the primary key defines a b-tree on the data in a way that allows comparison using any prefix of the key.